### ISSUE TITLE
editor/debugger ignoring `rorg` directive

### ISSUE THREAD

**Issue by langel** [sehugg/8bitworkshop#183]
I've been working with bank switching capable mappers 2 and 4 for a while. The assembler and the emulator operate as expected. But the editor is showing a rom address based on the .org directive and the debugger seems to be following that.

Here is an example repo for testing: https://github.com/langel/beat-em-high

I've been digging through the source and looks like `rorg` is missing from parseDirective() in /src/worker/assembler.ts 
...but that's my best guess (and I'm struggling to get github integration working on my local).

**Comment by sehugg** [comment]
Yeah, RORG support was developed for the 2600 and isn't great on the NES debugger. Do you still have a repo available?

**Comment by langel** [comment]
Sorry about that -- thought I already made that repo public

here: https://github.com/langel/beat-em-high
